### PR TITLE
Fix auth failure redirect

### PR DESCRIPTION
**Pull request by brandones**

### CODE REVIEW

File: packages/esm-api/jest.config.js [hunk 1]
```diff
@@ -6,8 +6,7 @@ module.exports = {
     "lodash-es": "lodash",
     "@openmrs/esm-error-handling":
       "<rootDir>/__mocks__/openmrs-esm-error-handling.mock.ts",
-    "@openmrs/esm-config":
-      "<rootDir>/__mocks__/openmrs-esm-module-config.mock.ts",
+    "@openmrs/esm-config": "<rootDir>/__mocks__/openmrs-esm-config.mock.ts",
```
Comment (FlorianRappl) [inline packages/esm-api/jest.config.js:9]: This was not committed. I only see that the ...module-config.mock.ts was removed, but I don't see the file added (or the original one renamed).

**Review by FlorianRappl — COMMENTED** [review]
I think this is a good change.

File: packages/esm-api/jest.config.js (hunk 1 above)
Comment (brandones) [reply packages/esm-api/jest.config.js:9]: Whoops! Good catch, thanks. Fixed.

**Review by FlorianRappl — APPROVED** [review]
LGTM - thanks :beers:!

**Pull request merged**